static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/dismiss-window/variant-2-day.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import type { CurrentABTest, SignInGateVariant } from '../../types';
+import { componentName } from '../../component';
+import {
+    hasUserDismissedGateInWindow,
+    isNPageOrHigherPageView,
+    isLoggedIn,
+    isInvalidArticleType,
+    isInvalidSection,
+    isIOS9,
+    setGatePageTargeting,
+} from '../../helper';
+
+// pull in the show method from the design folder, which has the html template and and click handlers etc.
+import { designShow } from '../design/main-variant';
+
+// define the variant name here
+const variant = 'dissmiss-window-variant-2-day';
+
+// method which returns a boolean determining if this variant can be shown on the current pageview
+const canShow: (name?: string) => boolean = (name = '') => {
+    const isGateDismissed = hasUserDismissedGateInWindow({
+        window: 'dev',
```

Comment:
should this be `day`?

Reply:
yes, it should. Was using dev mode (3min) for testing locally and in CODE. !!!  TODO